nss-day-cohort-29/reactive-nutshell-lochsea
# Nh events setup

# Description

added events to nav bar and routing template as well as initializing events files



## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)

# Testing Instructions for Change Made

`git fetch --all`
`git checkout nh-events-setup`
`npm start`
check to see if events is now on the nav bar

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings
- [ x] I have added test instructions that prove my fix is effective or that my feature works
